### ISSUE TITLE
Wrong dll name for netcore2.0

### ISSUE THREAD

**Issue by PulsarFX** [Bassman2/JenkinsWebApi#5]
All dlls in the nuget have the name `JenkinsWebApi.dll`.
Only in the `netcoreapp2.0` folder, the dll is named `JenkinsWebApiCore20.dll`

This causes the following exception:

```
ReflectionTypeLoadException:
'Unable to load one or more of the requested types.
Could not load file or assembly 'JenkinsWebApi, Version=2.2.0.0, Culture=neutral, PublicKeyToken=null'. Das System kann die angegebene Datei nicht finden.'

```

**Comment by PulsarFX** [comment]
maybe it's also an issue, because I call jenkins from a netstandard2.0 project and host it in a netcoreapp2.1

putting in a reference to `%userprofile%\.nuget\packages\jenkinswebapi\2.2.0\lib\netstandard2.0\JenkinsWebApi.dll` in both projects makes it run successfully.

**Comment by PulsarFX** [comment]
in `JenkinsWebApiCore20.csproj`
the following must be added:
```
 <TargetFramework>netcoreapp2.0</TargetFramework>
 <AssemblyName>JenkinsWebApi</AssemblyName>    <!--<-here-->
 <RootNamespace>JenkinsWebApi</RootNamespace>  <!--<-here-->
```

**Comment by Bassman2** [comment]
Fixed in V2.3
